packages/components/src/types/schema.ts

```diff
@@ -50,7 +50,7 @@ export const ArticlePageSchema = Type.Object({
 export const CollectionPageSchema = Type.Object({
   layout: Type.Literal(ISOMER_PAGE_LAYOUTS.Collection),
   page: CollectionPageMetaSchema,
-  content: Type.Array(Type.Null(), {
+  content: Type.Array(IsomerComponentsSchemas, {
```

Comment:
What's the issue with having it as null?

Reply:
Cos we would have two types, effectively `IsomerComponents[] | null`, then we need to separately validate in code that the layout is not `collection`, `file` or `link`. Standardising this means that we don't need to perform this validation in code ourselves and we can leave it to Ajv to perform the full validation for us.